config/servicemappings/apikeys.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: core.cnrm.cloud.google.com/v1alpha1
+kind: ServiceMapping
+metadata:
+  name: apikeys.cnrm.cloud.google.com
+  namespace: cnrm-system
+spec:
+  name: APIKeys
+  version: v1alpha1
```

Comment:
We don't have a service level version set to `v1alpha1` so there might be some rare feature gaps somewhere.

Meanwhile, we support all the other resources via autogen. So APIKeysKey configured here, I think there'll be two routes to promote a resource from alpha to beta. Feels like our code will get even more complicated.

Reply:
Well we're going to switch this one to direct actuation pretty fast, it looks like!  There are some significant gaps in the terraform implementation.